chart/values.yaml

```diff
@@ -29,6 +29,14 @@ env:
   # HCLOUD_NETWORK - see networking.enabled
   # ROBOT_ENABLED - see robot.enabled
 
+  # You can also use a file to provide secrets to the hcloud-cloud-controller-manager.
+  # This is currently possible for HCLOUD_TOKEN, ROBOT_USER, and ROBOT_PASSWORD.
+  # The value of the env var (e.g. HCLOUD_TOKEN) must start with "file:" to indicate that the value is a file path.
+  # The file must be provided externally (e.g. via secret injection).
+  # Example:
+  # HCLOUD_TOKEN:
+  #   value: "file:/etc/hetzner/token"
```

Comment:
Have you considered using a new env var name ? For example, using `HCLOUD_TOKEN_FILE` instead of using a `file:` prefix in the exiting variable? What are the pros and cons?

Reply:
Yes, I have thought about whether a new variable makes sense. I decided to use HCLOUD_TOKEN because it has been used everywhere and is therefore familiar to users. For the same reason, I can understand why you would want to use HCLOUD_TOKEN instead of HCLOUD_TOKEN_FILE. However, in my opinion, HCLOUD_TOKEN should be preferred, as it is currently the standard at Hetzner. 

You could also handle the whole thing like AWS, OpenStack etc, which have a standard location for credential files, but then this would still need to be defined, documented and built into as many Hetzner products as possible (e.g. /etc/hetzner/credentials, which could then contain the HCLOUD_TOKEN, ROBOT_USER, ROBOT_PASSWORD and perhaps other credentials as a JSON file), but I think this is not in the scope of this PR.

Reply:
Having a default discovery of the token+context that is used by all tools would be my dream, but we do not have that right now.

---

I prefer a separate variable `HCLOUD_TOKEN_FILE`. `HCLOUD_TOKEN` is already standardized across many tools as being the (plain text) token. Adding another format that is only supported in a single (or 2) tool adds to the confusion.

---

I think the prefix should be `file:/` to match URIs that already support file paths like `file://home/julian/` (ie. as implemented in browsers). Though that is not relevant if specified in a `_FILE` env variable.

Reply:
Thank you for your input, i will add a `HCLOUD_TOKEN_FILE` variable that contains the file path with the prefix `file:/`.

So an example value would be `HETZNER_TOKEN_FILE=file://etc/hetzner/token`

Reply:
I think in that case we can drop the `file:/` prefix

Reply:
Switched to `_FILE` env vars in newest commit.